packages/ui-inputs/src/buttons.stories.tsx

```diff
@@ -192,22 +235,25 @@ export const Workshop: Story = () => {
             <Button label={text("Label", "Label")}
                     isActive={boolean("Active", false)}
```

Comment:
please add an `in processing` knob for the two first buttons, default to `false`

Reply:
added